fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -2121,6 +2121,13 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true)
     public static int profile_info_reserved_num = 500;
 
+    /**
+     * Number of stream load profile infos reserved by `ProfileManager` for recently executed stream load and routine load task.
+     * Default value: 500
+     */
+    @ConfField(mutable = true)
+    public static int load_profile_info_reserved_num = 500;
```

Comment:
You can use the `profile_info_reserved_num`?

Reply:
Would it be better to control them separately，when I want to adjust the the size of load_profile, it won't affect query